cli/ceph/osd/pool.py

```diff
@@ -0,0 +1,122 @@
+from cli import Cli
+
+
+class Pool(Cli):
+    """This module provides CLI interface to manage the MGR service."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Pool, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} pool"
+
+    def rename(self, current_pool_name, new_pool_name):
```

Comment:
No need to mention `pool` keyword for variables here as class name itself classifies it.

Reply:
removed pool from all args in the class